patches/net/minecraft/client/particle/Particle.java.patch

```diff
@@ -1,15 +1,14 @@
 --- a/net/minecraft/client/particle/Particle.java
 +++ b/net/minecraft/client/particle/Particle.java
-@@ -245,6 +_,18 @@
+@@ -245,6 +_,17 @@
          return Optional.empty();
      }
  
 +    /**
-+     * Forge added method that controls if a particle should be culled to it's bounding box.
-+     * Default behaviour is culling enabled
++     * Returns the bounding box that should be used for particle culling.
```

Comment:
Please add a reference to the `IBlockEntityRendererExtension#INFINITE_EXTENT_AABB` for uncullable particles, especially since it's not in an obvious place (might even make sense to copy it over to `Particle` instead of re-using the existing one).